jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -38,6 +39,18 @@ def parse_data(app, file_obj, data_type=None, data_label=None, parent=None):
         The data type used to explicitly differentiate parsed data.
     data_label : str, optional
         The label to be applied to the Glue data component.
+    parent : str, optional
+        Data label for "parent" data to associate with the loaded data as "child".
+    cache : None, bool, or str
+        Cache the downloaded file if the data are retrieved by a query
+        to a URL or URI.
+    local_path : str, optional
+        Cache remote files to this path. This is only used if data is
+        requested from `astroquery.mast`.
+    timeout : float, optional
+        If downloading from a remote URL, set the timeout limit for
+        remote requests in seconds (passed to
+        `~astropy.utils.data.download_file`).
```

Comment:
astroquery also takes this, theoretically, unless you have checked that it doesn't?

Reply:
OK but that isn't reflected in the docstring here. Only `astropy.utils.data.download_file` is mentioned.